styleguide/source/_patterns/03-templates/02-article/article.twig

```diff
@@ -0,0 +1,50 @@
+    {% include 'molecules-figure' %}
+    {% include 'atoms-h3' with { 'content': 'Outreach Programs', 'id': 'section-3' }  %}
+    {% include 'atoms-paragraph' with { 'content': 'In luctus at ligula vitae venenatis. Sed ac nisi erat. Cras eleifend dolor sit amet risus laoreet lobortis. Nunc lectus nisl, consequat in risus quis, interdum tincidunt odio. Morbi pulvinar egestas ipsum, vel placerat dignissim leo, vitae maximus nisi turpis nec odio. Nam ullamcorper leo nec lobortis commodo. Donec elementum purus lorem, imperdiet sollicitudin dolor dictum eu.' } %}
+    {% include 'atoms-unordered' %}
+    {% include 'atoms-ordered' %}
+    {% include 'atoms-paragraph' with { 'content': 'Aliquam quis leo id orci maximus venenatis. Nam ut sem sed ultrices justo suscipit ac. Nam eifend enim eu turpis mi id nunc consequat, eu at, nulla non auctor grvida, turpis nisi vestib ulum quam, ac fermentum justo nunc id.' } %}
+    {# Add in time feature to video title #}
+    {% include 'molecules-figure-video' %}
+    {% include 'atoms-paragraph' with { 'content': 'In luctus at ligula vitae venenatis. Sed ac nisi erat. Cras eleifend dolor sit amet risus laoreet lobortis. Nunc lectus nisl, consequat in risus quis, interdum tincidunt odio. Morbi pulvinar egestas ipsum, vel placerat dignissim leo, vitae maximus nisi turpis nec odio. Nam ullamcorper leo nec lobortis commodo. Donec elementum purus lorem, imperdiet sollicitudin dolor dictum eu.' } %}
+    {% include 'molecules-pull-quote' %}
+    {% include 'atoms-h3' with { 'content': 'Lorem Ipsum', 'id': 'section-4' }  %}
+    {% include 'atoms-paragraph' with { 'content': 'In luctus at ligula vitae venenatis. Sed ac nisi erat. Cras eleifend dolor sit amet risus laoreet lobortis. Nunc lectus nisl, consequat in risus quis, interdum tincidunt odio. Morbi pulvinar egestas ipsum, vel placerat dignissim leo, vitae maximus nisi turpis nec odio. Nam ullamcorper leo nec lobortis commodo. Donec elementum purus lorem, imperdiet sollicitudin dolor dictum eu.' } %}
+    {% include 'molecules-inline-bio' %}
+    {% include 'atoms-paragraph' with { 'content': 'Sed massa augue, sollicitudin vel metus a, posuere scelerisque erat. In vulputate feugiat neque, sed aliquam risus. Cras blandit enim nec mauris egestas accumsan. Donec sed maximus justout pellentesque sem. Suspendisse semper varius turpis, vel congue risus mollis sit amet. Praesent molestie molestie ligula, ut viverra.' } %}
+  </div>
+  <div class="layout_two_up-heavy-right-offset-secondary">
+    {# Share this molecule goes here #}
+  </div>
+  <div class="layout_two_up-heavy-right-offset-primary">
+    {% include 'molecules-list-rule' %}
+    {% include 'molecules-list-rule-metadata' %}
+  </div>
+</div>
+<div class="layout layout-one-up">
+  {% include 'organisms-related-content' %}
```

Comment:
At about `934px` viewport the `.date-block_date` component type gets cutoff.
![screen shot 2017-03-06 at 10 22 17 am](https://cloud.githubusercontent.com/assets/362529/23618779/12077ff2-0257-11e7-8373-6b919daf7d1f.png)